sdk/src/test/scala/com/horizen/block/MainchainBlockReferenceTest.scala

```diff
@@ -963,6 +988,136 @@ class MainchainBlockReferenceTest extends JUnitSuite {
     assertTrue("Block must not contain transaction.", mcblock.data.sidechainRelatedAggregatedTransaction.isEmpty)
   }
 
+  @Test
+  def mcBlockReferenceDataSerializationMultipleCertificates(): Unit = {
```

Comment:
mcBlockReferenceDataSerializationMultipleCertificates -> mcBlockReferenceDataSerialization**Regression**MultipleCertificates

Let's specify that it is `regression`